src/helpers/proofs/generateERC721.ts

```diff
@@ -0,0 +1,53 @@
+import { ProofStore } from 'stores/ProofStore'
+import {
+  getEddsaPublicKey,
+  requestAddressOwnershipAttestation,
+  requestBalanceAttestation,
+} from 'helpers/proofs/attestor'
+import Network from '@big-whale-labs/stores/dist/models/Network'
+import buildERC721Proof from 'helpers/proofs/buildERC721Proof'
+import checkNavigator from 'helpers/proofs/checkNavigator'
+import getNullifierMessage from 'helpers/proofs/getNullifierMessage'
+import handleError from 'helpers/handleError'
+import walletStore from 'stores/WalletStore'
+
+export default async function generateERC721(
+  store: ProofStore,
+  original: string,
+  network: Network
+) {
+  try {
+    const account = walletStore.account
```

Comment:
btw, do we need to use capital `WalletStore` here?

Reply:
We use lowercase variables and uppercase classes, I think this is true for stores too